application/dash_application/views/single_lane_rna.py

```diff
@@ -122,15 +122,26 @@ def get_rna_data():
     #  contain multiple DataFrame/caches)
     rna_df = util.get_rnaseqqc()
 
+    # RNA-SeqQC does not correctly report total reads :(
+    # Use FastQC (summing R1 and R2) to get machine produced total reads
+    fq_total_reads = util.get_fastqc().groupby(
+        [util.FASTQC_COL.Run, util.FASTQC_COL.Lane, util.FASTQC_COL.Barcodes]
+    )[util.FASTQC_COL.TotalSequences].sum().to_frame().reset_index()
```

Comment:
I would appreciate a comment on how this works! Also, why 'util.FASTQC_COL' rather than importing FASTQC_COL into this module?